controllers/registration.go

```diff
@@ -0,0 +1,54 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/beego/beego/v2/server/web"
+	"google-scraper/forms"
+)
+
+type CommonTemplate struct {
+	web.Controller
+}
+
+func (c *CommonTemplate) Prepare() {
```

Comment:
As done by Hoangg, naming it `BaseController` or `baseTemplate` would be more standard since it's meant to be inherited/used in other objects.